File: W1/square2.c
#include <stdio.h>

int main() {
    int size;
    
    scanf("%d", &size);
    
    for ( int column = 0, counter = 1; column < size; column++ ) {
        for ( int row = 1; row < size; row++ ) {
            printf("%d ", counter);
            counter += 1;
        }
        printf("%d\n", counter);
        counter += 1;
    }
    
    return 0;
}

File: PHP/square2.php
<?php
    $size = fgets(STDIN);

    for ( $column = 0, $counter = 1; $column < $size; $column++ ) {
        for ( $row = 1; $row < $size; $row++ ) {
            echo $counter . ' ';
            $counter += 1;
        }
        echo $counter . PHP_EOL;
        $counter += 1;
    }
?>
